clusterloader2/pkg/prometheus/manifests/prometheus-rules.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: prometheus-k8s-rules
+  namespace: monitoring
+spec:
+  groups:
+  - name: kube-proxy.rules
+    rules:
+    - expr: |
+        histogram_quantile(0.99, sum(rate(kubeproxy_network_programming_latency_seconds_bucket[10m])) by (le))
+      record: kubeproxy:kubeproxy_network_programming_latency:histogram_quantile
```

Comment:
The other thing that is different than in the definition is that you make aggregation across all kube-proxies, whereas in definition we say:
```In default Kubernetes installation, 99th percentile of (99th percentiles across all programmers (e.g. iptables)) per cluster-day <= X```

So we take 99% of fastest kube-proxies and check those.

Maybe we should consider changing the definition - I wrote it that way because I wasn't sure if aggregation percentiles across all nodes is feasible, but it seems it is. Thoughts?

Reply:
Discussed personally -  https://github.com/kubernetes/community/pull/3307